docs/design/coding-conventions.md

```diff
@@ -314,13 +314,58 @@ Note that this also applies to uses of "ID" as an abbreviation for "identifier"
 
 Prefixes based on types (e.g., `p` for pointers) should never be used.
 
-Global variables should have a `g` prefix, e.g. `gCounter`.
-Non-`const` `static` class members can have an `s` prefix if that suits your fancy.
+Global variables should have a `g_` prefix, e.g. `g_counter`.
+Non-`const` `static` class members can have an `s_` prefix if that suits your fancy.
```

Comment:
Hmm... When you've been advocating for the `m_` prefix for members I assumed that was *just* for members, and you wouldn't want to mess with `g` as the prefix for globals.

I think this is going to be a stalemate, where I have a strong negative response to using underscores as part of an "ordinary" camel-cased name, and you seem to have a strong negative reaction to the "main" part of a name like `gCounter` starting with a capital if the convention is for variables to start with lower-case.

Reply:
It can be just for members - I'm just surprised that the consistency of having g and s work the same way isn't seen as a good thing. But hey ho - if that's the straw that breaks the camels back, then I can certainly live with that inconsistency.